api-docs/machine-insights-streaming-dataformat.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Machine Insights Streaming Dataformat
+category: 62fbabf8d9095e057cc1cd2c
+parentDocSlug: streaming-api
+---
+The Machine Insights based streaming dataformat provides high-level semantic information about machines. This means that e.g. information about "Operating Hours", "Fuel Remaining in Tank", "DEF Level" and "Battery Voltage" are immediately present in a normalized format - no matter the input source. This is perfectly tailored towards usage in a datalake or in AI-based analytics.
+
+> 📘 All datafields are optional
+>
+> For the same machine some might be filled out in one message while others are filled out in the next message; which fields are filled out depends on the technical details of the datacollection. The interpretation should always be that the individual datapoint was provided at that individual timestamp.
+
+Data is in general in-order, that is most recent datapoint last, but this is not guaranteed. If the most recent data is needed the timestamps should be compared. Any processing of historic data should be prepared to receive and handle data out of order.
```

Comment:
How about: "The data is generally ordered in sequence, with the most recent datapoint appearing last, although this order is not guaranteed." ?